docs/Release_Guide/metplotpy_bugfix.rst

```diff
@@ -5,8 +5,15 @@ METplotpy Bugfix Release
 
 Create a new vX.Y.Z bugfix release from the main_vX.Y branch.
 
+.. include:: release_steps/open_release_issue.rst
 .. include:: release_steps/clone_project_repository.rst
-.. include:: release_steps/metplotpy/update_version.rst
-.. include:: release_steps/metplotpy/update_docs.rst
+.. include:: release_steps/checkout_main_branch.rst
+.. include:: release_steps/metplotpy/update_version_bugfix.rst
 .. include:: release_steps/update_release_notes_bugfix.rst
+.. include:: release_steps/metplotpy/update_docs_bugfix.rst
+.. include:: release_steps/merge_release_issue.rst
+.. include:: release_steps/create_release_on_github.rst
+.. include:: release_steps/metplotpy/create_release_extra.rst
+.. include:: release_steps/metplotpy/update_dtc_website.rst
+.. include:: release_steps/finalize_release_on_github_official.rst
```

Comment:
```suggestion
.. include:: release_steps/finalize_release_on_github_bugfix.rst
```
Could you please update this to be the bugfix version instead of the official version?